Source/WTF/Scripts/generate-unified-source-bundles.rb

```diff
@@ -203,7 +203,7 @@ def initialize(extension, max)
         @extension = extension
         @fileCount = 0
         @bundleCount = 0
-        @currentBundleText = ""
+        @currentBundleText = "#pragma clang diagnostic ignored \"-Wunknown-pragmas\"\n"
```

Comment:
```suggestion
        @currentBundleText = "#pragma GCC diagnostic ignored \"-Wunknown-pragmas\"\n"
```

Remember that if you use `#pragma clang` then you need to guard it with `#if COMPILER(CLANG)` to ensure that GCC never sees it, whereas `#pragma GCC` is respected by both GCC and Clang.